connector/src/yang/connector/netconf.py

```diff
@@ -364,6 +366,12 @@ def connect(self):
             except Exception:
                 pass
 
+        # check timeout
+        try:
+            defaults['timeout'] = int(self.connection_info['timeout'])
```

Comment:
https://github.com/CiscoTestAutomation/yang/blob/master/connector/src/yang/connector/netconf.py#L266

`self.timeout` already exist via `__init__`. you might want to use this instead to be simple.

and please be aware the `timeout` is already used here. https://github.com/CiscoTestAutomation/yang/blob/master/connector/src/yang/connector/netconf.py#L284

so the timeout value will be shared between this and your code.